Clarify granular permissions opt-in is irreversible for read-only users (#9694)
fishtown-analytics/docs.getdbt.com: website/docs/docs/platform/manage-access/seats-and-users.md

If a user's IdP groups do not match _any_ license type mappings, <Constant name="dbt" /> assigns a Developer license by default.
:::

> ## Granular permissioning <Lifecycle status="managed,managed_plus"/>

Author: ```suggestion
## Granular permissioning <Lifecycle status="managed,managed_plus" />
```

<Constant name="dbt" /> Enterprise-tier plans support role-based access controls for configuring granular in-app permissions. See [access control](/docs/platform/manage-access/about-user-access) for more information on Enterprise permissioning.

> ### Read-only granular permissions

Reviewer: ```suggestion
### Read-Only granular permissions
```

Granular permissions allow you to control which projects your Read-Only users can access by assigning them to specific groups, rather than granting access to the entire account.

Enabling granular permissions is a one-time, permanent change. Before you enable this, make sure all your Read-Only users are added to their groups. If you don't do this first, those users may lose access to their projects.

> For complete setup instructions, refer to [Set up read-only user access](/docs/platform/manage-access/about-user-access#set-up-read-only-user-access) for the full setup steps.

Author: ```suggestion
For complete setup instructions, refer to [Set up read-only user access](/docs/platform/manage-access/about-access#enable-granular-permissions-for-read-only-users) for the full setup steps.
```

> For complete setup instructions, refer to [Set up read-only user access](/docs/platform/manage-access/about-user-access#enable-granular-permissions-for-read-only-users) for the full setup steps.

Reviewer: Hiya Mirna

Is this link correct correct - the link text says “Set up read-only user access” but the URL points to #enable-granular-permissions-for-read-only-users?

> For complete setup instructions, refer to [Set up read-only user access](/docs/platform/manage-access/about-user-access#enable-granular-permissions-for-read-only-users) for the full setup steps.

Author: ```suggestion
For complete setup instructions, refer to [How to enable granular permissions for read-only users](/docs/platform/manage-access/about-user-access#enable-granular-permissions-for-read-only-users) for the full setup steps.
```

## Granular permissioning

<Constant name="dbt" /> Enterprise-tier plans support role-based access controls for configuring granular in-app permissions. See [access control](/docs/platform/manage-access/about-user-access) for more information on Enterprise permissioning.

> :::caution Enabling granular permissions is irreversible

Author: ```suggestion
#### Read Only granular permissions
